libs/overlay/src/lib/ChatOverlay.ts

```diff
@@ -358,7 +360,7 @@ export class ChatOverlay {
    */
   destroy() {
     window.removeEventListener('message', this.process);
-    this.iframeInteraction.fail('Chat Overlay destroyed');
+    this.iframeInteraction.complete();
```

Comment:
Not sure that this is good idea. Actually, your iframe is not complete at this moment, but failing. Maybe we should just not log this as error, but still fail promise?
Moreover if this iframe interaction completed that means that user can send chat overlay options but this is not right scenario - they should do this only when it's really complete

Reply:
You are right. Reverted.
Added `try catch` block.